app/ui-react/packages/ui/src/Data/Virtualizations/Views/ViewEditContent.css

```diff
@@ -14,3 +14,8 @@
   margin-left: auto;
   width: 90%;
 }
+
+.CodeMirror-hints {
```

Comment:
Should put this in the `TextEditor` component's CSS too, probably be good to increase the specificity just to be on the safe side.

Reply:
moved and amended the PR